microsoft_intune/plugin.spec.yaml

```diff
@@ -0,0 +1,77 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: microsoft_intune
+title: "Microsoft Intune"
+description: Integrate Microsoft Intune into your workflow
+version: 1.0.0
+vendor: rapid7
+support: community
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/microsoft_intune
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://microsoft.com
+tags:
+  - microsoft
+  - intune
+  - azure
+hub_tags:
+  use_cases: [data_utility]
+  keywords: [intune, cloud, azure]
+  features: []
+connection:
+  credentials:
+    title: Credentials
+    description: E-mail address and password
+    type: credential_username_password
+    required: true
+    example: '{"username": "user@example.com", "password": "mypassword"}'
+  url:
+    title: Host URL
+    description: Base URL for the Microsoft endpoint
+    type: string
+    required: true
+    example: https://example.com
```

Comment:
I think the InTune documentation uses, https://management.azure.com for all customers

If there's a common cloud URL, we should use that here with a default.

e.g. Assuming this is the correct URL

```
default: https://management.azure.com
example: https://management.azure.com
```